hook_lib.py

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env python
```

Comment:
good catch!  Presumably some refactor removed main and forgot to clean this up.  Did you need to chmod?  Im assuming it had the binary bit given it had a shebang.